_positions/front-end-designer.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Front End Designer
```

Comment:
Not a blocker, but I would lean towards calling this "UI Designer" or "Web Designer".